docs/flashing.md

```diff
@@ -345,6 +345,40 @@ Flashing sequence:
 3. Flash a .bin file
 4. Reset the device into application mode (may be done automatically)
 
+## AT32 DFU
+
+All AT32 MCUs come preloaded with a factory bootloader that cannot be modified nor deleted.
+
+To ensure compatibility with the AT32-DFU bootloader, make sure this block is present in your `rules.mk`:
+
+```make
+# Bootloader selection
+BOOTLOADER = at32-dfu
+```
+
+Compatible flashers:
+
+* [QMK Toolbox](https://github.com/qmk/qmk_toolbox/releases) (recommended GUI)
```

Comment:
Toolbox looks for known pid:vid pairs when flashing, and no PR has been made to add support, so this statement here is incorrect.

Reply:
I've just removed, will added back when Toolbox supported.